### ISSUE TITLE
Import-Excel error count regression bug

### ISSUE THREAD

**Issue by DarkLite1** [dfinke/ImportExcel#1243]
## Description
Importing an Excel sheet that does not exist in the workbook when using `Import-Excel` results in 2 errors for version [7.4.2](https://github.com/dfinke/ImportExcel/archive/refs/tags/v7.4.2.zip) and 3 errors for version [7.5.0 and higher](https://github.com/dfinke/ImportExcel/archive/refs/tags/v7.5.0.zip). This brakes the error handling in one of our scripts.

```powershell
try {
    $params = @{
        Path          = 'C:\file.xlsx'
        WorksheetName = 'NonExisting'
    }
    Import-Excel @params
}
catch {
    # 7.4.2 : 2 errors
    # 7.5.0 : 3 errors
    $Error.Count
}
```

## Proposed solution
Correct the error handling of `Import-Excel` to only generate a single error with a descriptive text and keep internal errors of the module function inside the function.

## Actions already taken
We previously reported this issue in #1188 and provided a PR with a fix for this in #1190. Both the issue and the PR have been closed without further investigation or feedback.

## Root cause
The third error in version 7.5.0 and above comes from trying to access the non existing worksheet which results in the third error:
![image](https://user-images.githubusercontent.com/10828570/188409916-1c55bcfc-ad71-4a73-8a2a-67c0fb1394a1.png)

Looking at the code it is clearly visible that error handling is not properly set up:
![image](https://user-images.githubusercontent.com/10828570/188410087-856160ac-0536-48c1-b54e-71cb0a494645.png)

Thank you for having another look at this.

**Comment by DarkLite1** [comment]
Issue stil valid

**Comment by DarkLite1** [comment]
Good morning @dfinke 

I couldn't find the fix in [v7.8.2](https://github.com/dfinke/ImportExcel/compare/v7.8.1...v7.8.2). Is it fixed in another branch maybe?

**Comment by dfinke** [comment]
It is not fixed, and no plans to address it.

**Comment by DarkLite1** [comment]
Thanks for getting back to me. It's strange to see the first issue being closed as "_wontimplement_' and this one being closed as "_completed_" while it's not implemented nor further discussed. 

In the spirit of open source collaboration I find it a pity that my [PR](https://github.com/dfinke/ImportExcel/pull/1190) has not been accepted, nor was there a constructive discussion on proper error handling. If it was something I wrote, or if my words were wrongly interpreted, I'd like to offer my apologies as that was not my intention. I've tried to talk to you on chat but you did not respond, so I guess this is where it ends.

I wish you all the best with the project and hope it continues to thrive.
